config/database.yml

```diff
@@ -4,13 +4,13 @@ development:
   database: myflix_development
   host: localhost
   pool: 5
-  username: jdhang
-  password: test 
+  username: jhang
+  password:
 
 test:
   adapter: postgresql
   encoding: unicode
   database: myflix_test
   pool: 5
-  username: jdhang
-  password: test
\ No newline at end of file
+  username: jhang
+  password:
```

Comment:
Ah yes, I did question this when I was committing, it felt wrong, probably should have gone with my gut. Will remove and add a sample file. Thanks!